src/test/isolation/specs/udf-insert-deadlock.spec

```diff
@@ -10,8 +10,8 @@
 
 setup
 {
-	CREATE TABLE udf_dl_one (a int, b int) DISTRIBUTED BY (a);
```

Comment:
why remove `DISTRIBUTED BY (a)`

Reply:
you're right. this is unnecessary. reverted.